src/examples/core/burn-asset/cli.sh

```
@@ -0,0 +1,8 @@
+# Burn an NFT using the Metaplex CLI
+
+# Burn a single asset by its mint address
+mplx core asset burn <assetId>
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)